prime-dotnet-webapi/Controllers/LabtechsController.cs

```diff
@@ -0,0 +1,50 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Auth;
+using Prime.Models.Api;
+using Prime.Services;
+using Prime.HttpClients;
+using Prime.ViewModels.Parties;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/parties/[controller]")]
+    [ApiController]
+    [Authorize(Policy = Policies.User)]
+    public class LabtechsController : ControllerBase
+    {
+        private readonly IPartyService _partyService;
+        private readonly IKeycloakAdministrationClient _keycloakClient;
+
+        public LabtechsController(
+            IPartyService partyService,
+            IKeycloakAdministrationClient keycloakClient)
+        {
+            _partyService = partyService;
+            _keycloakClient = keycloakClient;
+        }
+
+        // POST: api/parties/labtechs
+        /// <summary>
+        /// Creates a new Labtech.
+        /// If successful, also updates Keycloak with additional user info and the Labtech role.
+        /// </summary>
+        [HttpPost(Name = nameof(CreateLabtech))]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<ActionResult> CreateLabtech(LabtechChangeModel labtech)
+        {
+            await _partyService.CreateOrUpdatePartyAsync(labtech, User);
+
```

Comment:
This seems to be working properly:  I had an existing `Party` record, signing authority for an organization, and that record was updated.  Can we have a comment that this is the design?  The only documentation I have that this is the design are a few chat messages in "The Tech Team".

Reply:
Yes, this is the design

Reply:
Is it worthwhile to put this explicitly as a comment in the source code?